src/app/members/[id]/profile/page.tsx

```diff
@@ -0,0 +1,18 @@
+import type { Metadata } from "next";
+import { MemberProfileContainer } from "@/components/ui/members/member-profile-container";
+import { Id } from "@/types/general";
+import { use } from "react";
+
+export const metadata: Metadata = {
+    title: "Member Profile",
+    description: "Member Profile page",
+};
+
+export default function ProfilePage({
+    params,
+}: {
+    params: Promise<{ id: Id }>;
+}) {
+    const userId = use(params).id;
+    return <MemberProfileContainer userId={userId} />;
```

Comment:
Just a reminder to wrap your `MemberProfileContainer` with `<PageContainer></PageContainer>` so it centers on the page and has consistent padding to the other pages.

Reply:
Done! e64d81d8484d4b7550856905ea73dfd23dabc75c